app/views/reporting/user_reports/index.html.erb

```diff
@@ -10,24 +10,45 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h2 class='govuk-heading-m'>Latest complete monthly reports<h2>
+    <h2 class='govuk-heading-m'>Latest complete reports<h2>
 
     <% @monthly_reports.each do |report|%>
       <h3 class='govuk-heading-s'>
         <%= govuk_link_to(
               report.title,
-              reporting_monthly_report_path(report_type: report.report_type, epoch: report.to_param),
+              reporting_temporal_report_path(
+                interval: 'monthly',
+                report_type: report.report_type,
+                period: report.to_param
+              ),
               { no_visited_state: true }
             )%>
       </h3>
     <% end %>
 
-    <h2 class='govuk-heading-m'>Latest complete weekly reports<h2>
     <% @weekly_reports.each do |report|%>
       <h3 class='govuk-heading-s'>
         <%= govuk_link_to(
               report.title,
-              reporting_weekly_report_path(report_type: report.report_type, epoch: report.to_param),
+              reporting_temporal_report_path(
+                interval: 'weekly',
```

Comment:
Small thing but this could probably be a partial rather than two similar groups of code.